### ISSUE TITLE
[Unit 2] (high) Ideas for Assessment

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-curricula#64]
[Ideas for Assessment](https://deploy-preview-50--wai-curricula.netlify.com/curricula/1-2/#ideas-for-assessment)

> * Learners should be able to explain different types of accessibility features, such as colors with good contrast, types of barriers, and types of assistive tools and adaptive strategies.  
**Note**: Emphasize how accessibility is about enhancing the ability of people to use technology.
> * Learners should be able to demonstrate what they have learned from exploring assistive tools and adaptive strategies, such as explaining different settings they found in their web browser.
> * Learners should be able to describe how some adaptive strategies and assistive tools may benefit other users as well, such as aging individuals.

These don't see right -- but my brain has turned to mush. Sorry, I'll try to give better feedback soon.

**Comment by daniel-montalvo** [comment]
Further work is at 4953e64645ccbc8217695bc6155890273536915e#diff-5284d8637d198873f7517be26f7b72c9

**Comment by daniel-montalvo** [comment]
The section now looks as follows:

> ## Ideas for Assessment

>  Students should be able to:

>  * explain different types of accessibility features, and barriers
>  * list some assistive technologies and adaptive strategies.  
>  * perform basic tasks with assistive technologies, such as using the tab key to navigate different interfaces or using screen readers' navigation quick keys
>  * describe how some adaptive strategies may benefit other users as well, such as aging individuals.
>  * describe some of the components of web accessibility.

**Comment by shawna-slh** [comment]
Much clearer!

I think can just add to changelog and don't need to bring up for EOWG discussion.
